package.json

```diff
@@ -21,11 +21,12 @@
   "author": "Apiary Czech Republic, s.r.o. <support@apiary.io>",
   "license": "MIT",
   "dependencies": {
+    "dompurify": "^1.0.10",
+    "jsdom": "^14.0.0",
```

Comment:
`jsdom` should only be a devDependency used in tests. In browsers this should use the browser DOM.

Reply:
changed